src/sass/_instance_detail_page.scss

```diff
@@ -8,8 +8,10 @@
   }
 }
 
+.duplicate-instances-modal,
 .create-image-from-instance-modal {
   .p-modal__dialog {
+    justify-self: center;
```

Comment:
Why is the justify-self needed?

Reply:
The modal is not centered when someone opens something like the inspect-tab, vertically on the page, but it is now :) 
![image](https://github.com/user-attachments/assets/01c7978a-7662-4916-bb93-f6afaf5631db)

Reply:
It was always centred, wasn't it? Chrome dev tools report this as ignored:
![image](https://github.com/user-attachments/assets/db684697-8632-4bcb-a6a6-05922b0b1f69)
I am confused.

Reply:
The default behaviour for `.p-modal__dialog` from vanilla for smaller screen sizes should have the dialog stretch across 100vw with some padding.
![Screenshot from 2024-08-22 08-54-10](https://github.com/user-attachments/assets/d5215a63-1ccd-436a-8211-f66da1490436)
The style override here sets a fixed width for `.p-modal__dialog` so for smaller screen sizes the modal would be off center.
![Screenshot from 2024-08-22 08-53-30](https://github.com/user-attachments/assets/2b71cfb1-0c5a-4ede-934c-4326c0d6e2c3)

I think either we override the modal width only for large screen size:
```
.p-modal__dialog {
    @include large{
      width: 35rem;
    }
  }
```
Or we can set margin to auto so that the modal will be centered for smaller screen sizes:
```
.p-modal__dialog {
    margin: auto;
    width: 35rem;
  }
```
The first approach might be preferable I rate?

It is however quite interesting that the browser shows `justify-self` not having an effect since removing the styling does revert the modal to what was shown in the first screenshot :thinking:

Reply:
+1 on limiting the size only for large screens